guides/Browser_History_Embedding_with_ChromaDB.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Browser History Embedding and Search with ChromaDB"
+description: "A detailed guide on embedding and searching browser history using ChromaDB with support for both local and Azure embeddings."
+date: 2024-08-19
+author: "Jacob Gaffke"
```

Comment:
You have literally copied @giraffekey PR and made some changes on it?

Reply:
check the content! I don't know who's name i have to write on author so i have used from giraffeket PR. That's all i have done from PR> can check the content both has different way to guide!

Reply:
@nkkko this is first time I am doing on this so i don't know who author to be! If it still look like copied please let me know i will withdraw PR.

Reply:
yes, I will keep that in mind. Thanks for guide.